pdc/apps/auth/__init__.py

```diff
@@ -16,7 +16,7 @@
 Configuring settings.py
 -----------------------
 
-    MIDDLEWARE_CLASSES = (
+    MIDDLEWARE = (
         ...
```

Comment:
Do we need to change the MIDDLEWARE_CLASSES to MIDDLEWARE in settings.py?

Reply:
Nice find. Fixed.